hack/local-development/dev-setup-register-gardener

```diff
@@ -308,7 +310,7 @@ $ADMISSION_CONTROLLER_PORT_STRING
     service:
       namespace: garden
       name: gardener-admission-controller
-      port:
+      port: 2721
```

Comment:
why did you add the port specifically? Isn't `$ADMISSION_CONTROLLER_PORT_STRING`  adding the port to the path?